cql2pgjson/src/main/java/org/folio/cql2pgjson/util/Cql2SqlUtil.java

```diff
@@ -24,16 +24,28 @@
   private Cql2SqlUtil() {
     throw new UnsupportedOperationException("Cannot instantiate utility class.");
   }
-
   /**
    * Convert a CQL string to an SQL LIKE string.
-   * CQL escapes * ? ^ \ and SQL LIKE escapes \ % _.
+   * Signal the underlying function that this is not a quoted String case.
    *
    * @param s  CQL string without leading or trailing double quote
    * @return SQL LIKE string without leading or trailing single quote
    */
   @SuppressWarnings("squid:S3776")  // suppress "Cognitive Complexity of methods should not be too high"
   public static String cql2like(String s) {
+    return cql2String(s,false);
+  
+  }
+  /**
+   * Convert a CQL string to an SQL string.
+   * CQL escapes * ? ^ \ and SQL LIKE escapes \ % _.
```

Comment:
The comment about SQL LIKE should be moved to the sql2like method.

Here the comment should be "and SQL string constant escapes only the single quote ' by doubling it."